neoexchange/core/management/commands/lightcurve_extraction.py

```diff
@@ -474,18 +478,16 @@ def handle(self, *args, **options):
                         movie_file = make_gif(frames_list, sort=False, init_fr=100, center=3, out_path=out_path, plot_source=True,
                                               target_data=frame_data, show_reticle=True, progress=True)
                         if "WARNING" not in movie_file:
+                            # Create DataProduct
+                            save_dataproduct(obj=block, filepath=movie_file, filetype=DataProduct.GUIDER_GIF)
                             output_file_list.append('{},{}'.format(movie_file, data_path.lstrip(out_path)))
                             self.stdout.write("New gif created: {}".format(movie_file))
                         else:
                             self.stdout.write(movie_file)
-        alcdef_file.close()
+        save_dataproduct(obj=block.body, filepath=alcdef_file.name, filetype=DataProduct.ALCDEF_TXT, filename=alcdef_filename)
```

Comment:
This is problematic. `Lightcurve_extraction` can pull a series of superblocks all for the same object.
It runs through every block of every Super block and builds an ALCDEF file from all of them. 
It would be ideal to tie multiple blocks, or multiple super blocks to the single ALCDEF file, but if we MUST tie only a body to it, this should probably be `save_dataproduct(obj=start_super_block.body, filepath=alcdef_file.name, filetype=DataProduct.ALCDEF_TXT, filename=alcdef_filename)` since `block` is defined inside a nested for loop above.

Another solution would be to move the creation of ALCDEF files higher in the logic structure so that only a single Superblock was ever involved and tie the content_type to the superblock.